### DISCUSSION TITLE
Hello, people. I've never introduced myself

### DISCUSSION THREAD

**Discussion by marknkw** [community/community#48532] [New to GitHub]
I've used to work as a jr python data engineer. Now I'm without a job, but pursuing a new one eagerly.
Don't normally talk to anyone, so it seemed right to do it now.
How are you doing, ppl?

**Comment by melvincwng** [comment]
Hey @marknkw, welcome to GitHub Community 👋 

For starters, you can also introduce yourself in this [discussion thread](https://github.com/community/community/discussions/23204).

Have a good day ahead 😃
